asab/zookeeper/container.py

```diff
@@ -94,9 +95,19 @@ async def _do_advertise(self, zoocontainer):
 				await zoocontainer.ZooKeeper.set_data(self.Node, self.Data)
 				return
 
-			self.Node = await zoocontainer.ZooKeeper.create(
-				self.Path,
-				data=self.Data,
-				sequential=True,
-				ephemeral=True
-			)
+			try:
+				self.Node = await zoocontainer.ZooKeeper.create(
+					self.Path,
+					data=self.Data,
+					sequential=True,
+					ephemeral=True
+				)
+			except aiozk.exc.NoNode:
+				missing_node = self.Path.rstrip(self.Path.split("/")[-1])
+				await zoocontainer.ZooKeeper.create(missing_node)
```

Comment:
This is good ... but it has to be a `while` loop - `zoocontainer.ZooKeeper.create` should not be repeated as a call.
alternativelly, call `await zoocontainer.ZooKeeper.create(missing_node)` everytime.